extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js

```diff
@@ -66,6 +66,7 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 			'bidder_8': transformBidderPrice('veles', realSlotPrices, slotPricesIgnoringTimeout),
 			'bidder_9': transformBidderPrice('openx', realSlotPrices, slotPricesIgnoringTimeout),
 			'bidder_10': transformBidderPrice('appnexusAst', realSlotPrices, slotPricesIgnoringTimeout),
+			'bidder_11': transformBidderPrice('rubiconDisplay', realSlotPrices, slotPricesIgnoringTimeout),
```

Comment:
Is kikimora ready for this new field?
https://github.com/Wikia/crossbow/blob/master/dags/winter_wind/analytics-wikia/fastly/special/fact_adengadinfo_events/fact_adengadinfo_events_insert.sql